packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1334,3 +1334,57 @@ function VM-Clean-Up {
     Write-Host "[+] Clearing up free space. This may take a few minutes..." -ForegroundColor Green
     VM-Clear-FreeSpace
 }
+
+function VM-Add-To-Path {
+    param (
+        [string]$pathToAdd
+    )
+
+    # Function to normalize a path and handle exceptions
+    function NormalizePath {
+        param (
+            [string]$path
+        )
+        try {
+            return [System.IO.Path]::GetFullPath($path)
+        }
+        catch {
+            return $null
+        }
+    }
+
+    # Ensure the path to add is not null or empty
+    if ([string]::IsNullOrWhiteSpace($pathToAdd)) {
+        Write-Host "Invalid path provided."
+        return
+    }
+
+    # Get the Machine Path environment variable and split it into an array
+    $currentPaths = [System.Environment]::GetEnvironmentVariable("Path", [System.EnvironmentVariableTarget]::Machine) -split ';'
+
+    # Normalize the input path
+    $normalizedPathToAdd = NormalizePath $pathToAdd
+    if (-not $normalizedPathToAdd) {
+        Write-Host "Invalid path provided."
```

Comment:
I think we need to add more context in this message. I also think we should use `VM-Write-Log` instead of `Write-Host`
```suggestion
        VM-Write-Log "ERROR" "Tried to add invalid path to the Path: $pathToAdd"
```